### PR TITLE
Error aside effect

### PR DESCRIPTION
**Pull request by PawelOwiesek**
Add component Error with styles and formated App.js code.

### CODE REVIEW

File: src/features/AsideActions/styled.js [hunk 1]
```diff
@@ -22,4 +22,46 @@ export const Image = styled(image)`
...
+export const Button = styled.button`
+  display: flex;
+  align-items: center;
+  border: none;
+  background-color: ${({ theme }) => theme.color.scienceBlue};
+  padding: 16px 24px;
+  border-radius: 5px;
+`;
+
+export const Content = styled.p`
```
Comment (MirekSobota) [inline src/features/AsideActions/styled.js:47]: I think you need to use here different font:
`font-family: Open Sans;`

File: src/features/AsideActions/styled.js [hunk 2]
```diff
@@ -22,4 +22,46 @@ export const Image = styled(image)`
...
+export const Warning = styled(warningIcon)`
+  display: block;
+  margin: 0 auto;
+`;
+
+export const Title = styled.h2`
+  font-size: 36px;
+  color: ${({ theme }) => theme.color.woodsmoke};
+  font-weight: 600;
+  line-height: 120%;
```
Comment (MirekSobota) [inline src/features/AsideActions/styled.js:34]: Pleas change line-height to 1,2 :D

File: src/features/AsideActions/styled.js [hunk 3]
```diff
@@ -22,4 +22,46 @@ export const Image = styled(image)`
   align-self: center;
 `;
 
-export const Warning = styled(warningIcon)``;
+export const Warning = styled(warningIcon)`
+  display: block;
+  margin: 0 auto;
```
Comment (MirekSobota) [inline src/features/AsideActions/styled.js:27]: We need more space (180px) betwen Navigation and warningIcon, maybe you can do it like this:
`margin: 180px auto 24px;`

File: src/features/AsideActions/Actions/error.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { Warning, Container, Title, Info, Button, Content } from "../styled";
+
+const Error = () => (
```
Comment (MirekSobota) [inline src/features/AsideActions/Actions/error.js:3]: Could you please use `export const...` insted of `export default...` so we all use the same convention.

**Review by MirekSobota — APPROVED** [review]
I have added some comments, not big issues. Overall, it looks good.

File: src/features/AsideActions/styled.js (hunk 1 above)
Comment (PawelOwiesek) [reply src/features/AsideActions/styled.js:47]: You're right, I didn't notice that.

**Review by DorotaZielinska — APPROVED** [review]

**Review by DorotaZielinska — APPROVED** [review]

**Review by DorotaZielinska — APPROVED** [review]

**Review by DorotaZielinska — APPROVED** [review]

**Review by DorotaZielinska — APPROVED** [review]

**Review by DorotaZielinska — APPROVED** [review]

**Pull request merged**